.github/workflows/publish-docs.yml

```
@@ -90,18 +81,24 @@ jobs:
+      - name: Link Vercel Project
+        working-directory: docs/site
+        env:
+            VERCEL_PROJECT_ID: ${{ secrets.VERCEL_DOC_PROJECT_ID}}
+            VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+        run: vercel link --yes --project $VERCEL_PROJECT_ID --token $VERCEL_TOKEN
       - name: Deploy to Vercel
-        working-directory: docs
+        working-directory: docs/site
         env:
-          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+            VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 12

(indentation)